manta-trusted-setup/src/groth16/ceremony/mod.rs

```diff
@@ -194,8 +194,8 @@ where
         match self {
             Self::NotRegistered => write!(
                 f,
-                "Registry update is taking longer than expected. \
-                 Please make sure you have submitted your registration form and try again later.",
+                "Please make sure you have submitted your registration form. \
+                 Check whether the ceremony has begun at <ceremony.manta.network>.",
```

Comment:
This is consistent with the other address scheme above:

```suggestion
                 Check whether the ceremony has begun at https://ceremony.manta.network.",
```